### PR TITLE
GEOPY-923: Add content for all EM loop configurations

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-923 - Update EM survey docs**

### CODE REVIEW

File: docs/content/geoh5_format/analyst/objects.rst [hunk 1]
```diff
@@ -469,145 +460,327 @@ Datasets
...
+    - "Unit": ``string``
+        Sampling units, must be one of ""Hertz (Hz)", "KiloHertz (kHz)",
+        "MegaHertz (MHz)" or "Gigahertz (GHz)".
+    - "Crossline offset property" ``uuid`` OR  "Crossline offset value" ``double``:
+        Offline offset between the receivers and transmitters,
+        either defined locally on vertices as a ``property`` OR globally as a constant ``value``.
+    - "Inline offset property" ``uuid`` OR  "Crossline offset value" ``double``:
+        Inline offset between the receivers and transmitters,
+        either defined locally on vertices as a ``property`` OR globally as a constant ``value``.
+    - "Inline offset property" ``uuid`` OR  "Crossline offset value" ``double``:
```
[the remark this answers is not in the document]
Comment (jamiefb) [reply docs/content/geoh5_format/analyst/objects.rst:639]: this also shows up in a couple places

**Review by jamiefb — CHANGES_REQUESTED** [review]
a couple small things

File: docs/content/geoh5_format/analyst/objects.rst [hunk 2]
```diff
@@ -469,145 +460,327 @@ Datasets
...
-**UUID : {41018a45-01a0-4c61-a7cb-9f32d8159df4}**
+:Metadata: json formatted ``string``
 
-*Not yet geoh5py implemented*
+    Dictionary of survey parameters shared with the :ref:`Transmitters <geoh5_afem_tx>`. The following items are core
+    parameters stored under the "EM Dataset" key.
 
-*To be further documented*
+    - "Channels": ``list`` of ``double``
+        Frequency channels at which data are recorder.
```
[the remark this answers is not in the document]
Comment (domfournier) [reply docs/content/geoh5_format/analyst/objects.rst]: Done

File: docs/content/geoh5_format/analyst/objects.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (domfournier) [reply docs/content/geoh5_format/analyst/objects.rst]: Done

**Review by jamiefb — APPROVED** [review]

**Pull request merged**